docs/site/sitemap.tsv

```diff
@@ -7,15 +7,7 @@
 #	If an item has 'sub items' it becomes a directory parent.		
 #	There must be a single 'index' entry here at the root.		
 index			
-	index/cldr-spec		
-		index/cldr-spec/core-data-for-new-locales	
-		index/cldr-spec/coverage-levels	
-		index/cldr-spec/collation-guidelines	
-		index/cldr-spec/picking-the-right-language-code	
-		index/cldr-spec/plural-rules	
-		index/cldr-spec/transliteration-guidelines	
-		index/cldr-spec/definitions	
-		index/bcp47-extension	
+	requesting_changes		
 	index/downloads		
```

Comment:
I think it would be better to have index/downloads before requesting changes

Reply:
I kind of expect most people to follow the news item to the latest release instead of the nav bar but I could be wrong.

Okay, so this ordering?
- CLDR Releases
- CLDR Spec
- CLDR Charts
- Requesting changes
- Acknowledgements
- General Information
- Information Hub for Linguists
- CLDR Technical Committee
- Schedule
- Internal Development (I'm not sure why we have this from the front page instead of a sub section of the TC)
- Archived Pages (I'm not sure why we have this from the front page, but I'm not sure where it would fit)
- CLDR Site Map

Reply:
I think that works.

On schedule, we might put that right below General info

- For internal development, it is handy for us to have it at the top level
- For archived pages, it's hard to know where else to put it. They are mostly (but I think not all) internal development.

Note: we could also have Steven extend the site map tooling to allow for the following, if we want more flexibility.
- Having pages in multiple locations
- Hiding pages (we want to make that explicit, so we don't inadvertantly skip a page).

Reply:
SG! Let me put together an updated commit for review.